tests/test_f.py

```diff
@@ -288,6 +288,51 @@ def test_join_bad_source():
     assert s.parts == s.flatten().parts == ("a", ",", "b", ",", "c")
 
 
+def test_preserved_join_empty():
+    s = F(" ").preserved_join([])
+    assert s == F("", parts=("",))
+
+
+def test_preserved_join_list():
+    b = 2
+    s = F(" ").preserved_join(["a", F(f"b={b}"), "c"])
+    assert s == "a b=2 c"
+    assert s.parts == (
+        "a",
+        " ",
+        "b=",
```

Comment:
I think it makes sense to want to be able to get to these literal parts and not be stuck with things like `FValue(source="(strings)[1]", value="b", formatted="b")`, but I think that should only happen after flattening, and this should probably still be `F(f"b={b}")`.

Reply:
(or an FValue containing that)

Reply:
Ahh I see what you mean. Ok done!